roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -106,7 +106,7 @@
     "compute_extension:admin_actions:resetNetwork": "rule:admin_or_owner",
     "compute_extension:admin_actions:injectNetworkInfo": "rule:admin_or_projAdmin_or_cloudAdmin",
     "compute_extension:admin_actions:createBackup": "rule:admin_or_owner",
-    "compute_extension:admin_actions:migrateLive": "rule:admin_api",
```

Comment:
This is to control if related action is shown in horizon. I think we need this for cloud_admin